### PR TITLE
fix(xvla): remove duplicated preprocessor steps in LIBERO env processor

### LINKED ISSUE #2955: Duplicated Steps in Environment and Policy PreProcessor when evaluating X-VLA on LIBERO
### Ticket Type

🐛 Bug Report (Something isn't working)

### Environment & System Info

```Shell

```

### Description

Note that this issue coincides heavily with #2954.

I identified a redundancy in the preprocessing pipeline when evaluating the **XVLA policy** within the **LIBERO environment**. 

The pipeline instantiates two processor chains:
1. **Environment Preprocessor:** via `make_env_pre_post_processors` → `make_xvla_libero_pre_post_processors`
2. **Policy Preprocessor:** via `make_pre_post_processors` → `make_xvla_pre_post_processors`

Both chains currently append `XVLAImageNetNormalizeProcessorStep` and `XVLAAddDomainIdProcessorStep`. Because the pipeline executes `Env Preprocessor` → `Policy Preprocessor`, these transformations are applied twice to the input observations.

### Reproduction / Code Analysis
The duplication occurs in `lerobot/policies/xvla/processor_xvla.py`.

**1. Environment-Specific Processor (`make_xvla_libero_pre_post_processors`)**
Adds normalization and domain ID:
```python
def make_xvla_libero_pre_post_processors() -> tuple[...]:
    # ...
    pre_processor_steps.extend(
        [LiberoProcessorStep(), XVLAImageNetNormalizeProcessorStep(), XVLAAddDomainIdProcessorStep()]
    )
    # ...
```

2. General Policy Processor (make_xvla_pre_post_processors) Also adds normalization and domain ID:
```python
def make_xvla_pre_post_processors(...) -> tuple[...]:
    # ...
    input_steps = [
        # ...
        XVLAImageToFloatProcessorStep(),
        XVLAImageNetNormalizeProcessorStep(), # DUPLICATE
        XVLAAddDomainIdProcessorStep(),       # DUPLICATE
        # ...
    ]
```
This leads to double normalization which in turns leads to erroneous input values and redundant processing. 

### Suggested Fix
Remove the general XVLA steps from the environment-specific processor (make_xvla_libero_pre_post_processors). It should likely only handle LiberoProcessorStep and leave the standard normalization to the policy processor.

```python
def make_xvla_libero_pre_post_processors() -> tuple[...]:
    # ...
    # Removed XVLAImageNetNormalizeProcessorStep and XVLAAddDomainIdProcessorStep
    pre_processor_steps.extend(
        [LiberoProcessorStep()]
    )
    # ...
```

### Additional Remarks
Currently this issue is mitigated when directly evaluating the pre-trained lerobot/xvla-libero checkpoint available on huggingface as the checkpoint there does not include the `XVLAImageToFloatProcessorStep` and the `XVLAImageNetNormalizeProcessorStep` (which I think is incorrect, see #2954).
When I first finetuned lerobot/xvla-libero on LIBERO with the additional `XVLAImageToFloatProcessorStep` and the `XVLAImageNetNormalizeProcessorStep`  steps in the preprocessing pipeline and then evaluated this model I ran into the double normalization error which led to negative input values.

### Context & Reproduction

_No response_

### Relevant logs or stack trace

```Shell

```

### Checklist

- [x] I have searched existing tickets to ensure this isn't a duplicate.
- [x] I am using the latest version of the `main` branch.
- [x] I have verified this is not an environment-specific problem.

### Additional Info / Workarounds

_No response_

### PR DESCRIPTION
**Pull request by wadeKeith**
Fixes #2955.

## Problem

In `processor_xvla.py`, both `make_xvla_pre_post_processors` (policy preprocessor) and `make_xvla_libero_pre_post_processors` (LIBERO environment preprocessor) added `XVLAImageNetNormalizeProcessorStep()` and `XVLAAddDomainIdProcessorStep()`.

During LIBERO evaluation, the pipeline runs: **Env Preprocessor → Policy Preprocessor**, which means ImageNet normalization and domain ID injection were applied **twice**.

## Fix

Removed `XVLAImageNetNormalizeProcessorStep()` and `XVLAAddDomainIdProcessorStep()` from `make_xvla_libero_pre_post_processors`, since they are already handled by the policy preprocessor (`make_xvla_pre_post_processors`, lines 73-74).

The LIBERO environment preprocessor now only contains `LiberoProcessorStep()`, which handles the LIBERO-specific observation formatting.

### CODE REVIEW

File: src/lerobot/policies/xvla/processor_xvla.py [hunk 1]
```diff
@@ -543,7 +543,7 @@ def make_xvla_libero_pre_post_processors() -> tuple[
     pre_processor_steps: list[ProcessorStep] = []
     post_processor_steps: list[ProcessorStep] = []
     pre_processor_steps.extend(
-        [LiberoProcessorStep(), XVLAImageNetNormalizeProcessorStep(), XVLAAddDomainIdProcessorStep()]
+        [LiberoProcessorStep()]
     )
```
[the remark this answers is not in the document]
Comment (wadeKeith) [reply src/lerobot/policies/xvla/processor_xvla.py:549]: Good catch on the concern. I verified that `make_xvla_pre_post_processors()` (called via `make_pre_post_processors()` in `factory.py:397`) **always** hardcodes both `XVLAImageNetNormalizeProcessorStep()` and `XVLAAddDomainIdProcessorStep()` in its input pipeline (processor_xvla.py lines 73-74). These steps are not loaded from checkpoint config — they're constructed at runtime from the policy class itself. So the policy preprocessor will always include them.

[the remark this answers is not in the document]
Comment (wadeKeith) [reply src/lerobot/policies/xvla/processor_xvla.py:549]: Agreed, a regression test would be valuable. I'll consider adding one in a follow-up if the maintainers want it as part of this PR.

**Comment by wadeKeith** [comment]
Hi maintainers 👋 Following up on the review discussion — I'll add a regression test to verify that ImageNet normalization and domain_id steps are applied exactly once through the XVLA+LIBERO pipeline. Will push the update shortly. Thanks for the feedback!

**Comment by wadeKeith** [comment]
👋 Friendly ping for review — this PR is ready (branch synced with main). Thanks!

**Comment by wadeKeith** [comment]
Friendly ping for review — CI is passing. Thanks!

**Comment by wadeKeith** [comment]
Friendly ping for review. Thanks!
